### ISSUE TITLE
Can JAWS add < mark > element recognition?

### ISSUE THREAD

**Issue by gretayharding** [FreedomScientific/standards-support#780]
### Summary
Highlighted text content that has been marked up with the < mark > element is not announced by JAWS. 

### Expected result
When using JAWS screenreader, the software should announce that the text is "marked."

### Actual result
JAWS does not recognize < mark >

### Example 1 – Simple Text
https://codepen.io/gretayharding/pen/PoVmzrP

### Example 2 – Math
Math can be complex to parse. Google provides tools to help students understand complex equations using highlighted text (to be updated with < mark >). Type this prompt into Google Search to experience how highlighted text is used to help students grok information. 
 
**Copy and paste the following into Google Search**
"If an object accelerates from rest, with a constant velocity 5.2m/s2, what will its velocity be at 28s"

### Screen readers that detect < mark >
- Chrome VOX
- NVDA
- Microsoft Narrator

### Screen readers that don't detect < mark >
- JAWS
- VoiceOver

#### Browser and version:
not applicable

### Additional Information
Currently screen reader users are unable to know if text is highlighted within Google Search results. As popularity of this text treatment expands among web content, the equitable experience gap widens, excluding users who are unable to perceive highlighted text.

Google's Search Accessibility team is proposing adding < mark > to highlighted text but there is one challenge – currently JAWS and VoiceOver do not recognize < mark >. 

**Would it be possible to add < mark > to JAWS?**

**Comment by stevefaulkner** [comment]
@gretayharding when i tested this recently JAWS did announce `<mark>` Tested again tonight [test case](https://cdpn.io/pen/debug/abjVGqY)

further reading: [Screen Readers support for text level HTML semantics](https://www.tpgi.com/screen-readers-support-for-text-level-html-semantics/)
